react/experiments/src/containers/ExperimentsList.js

```diff
@@ -363,9 +363,28 @@ class ExperimentsList extends React.Component {
                                 </div>
                                 <div className="table-setting__manage table-setting__column4">
                                     <div className="table-setting__items">
+                                        {(window.location.pathname  === '/en/board') ? (
```

Comment:
A not good solution to detect what we're showing: experiments or feature flags at all. Url could be changed. And how you will test it locally what app will be on your localhost:2321 without any path in the URL? 

In case of reading at least a bit up you will see "window.mode", but it is still not the best solution. Try to detect what you're showing with backend -- i hope there is a difference exists, if not - use window.mode.